src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -80,7 +80,8 @@ private function joinRelations(QueryBuilder $queryBuilder, string $resourceClass
                 continue;
             }
 
-            $method = false === $mapping['joinColumns'][0]['nullable'] ? 'innerJoin' : 'leftJoin';
+            $joinColumns = $mapping['joinColumns'] ?? $mapping['joinTable']['joinColumns'] ?? null;
+            $method = null === $joinColumns ? 'leftJoin' : (false === $joinColumns[0]['nullable'] ? 'innerJoin' : 'leftJoin');
```

Comment:
Nested ternary operators are very hard to read. Can you change change it to the following?

```php
$joinColumns = $mapping['joinColumns'] ?? $mapping['joinTable']['joinColumns'] ?? null;
if (null === $joinColumns) {
    $method = 'leftJoin';
} else {
    $method = false === $joinColumns[0]['nullable'] ? 'innerJoin' : 'leftJoin';
}
```

Thanks!